Tools/clinic/clinic.py

```diff
@@ -1954,27 +1954,32 @@ def dump(self):
         return "".join(texts)
 
 
+@dc.dataclass(slots=True, repr=False)
 class Destination:
-    def __init__(self, name, type, clinic, *args):
-        self.name = name
-        self.type = type
-        self.clinic = clinic
-        self.buffers = BufferSeries()
+    name: str
+    type: str
+    clinic: Clinic
+    buffers: BufferSeries = dc.field(init=False, default_factory=BufferSeries)
+    filename: str = dc.field(init=False)  # set in __post_init__
 
+    args: dc.InitVar[tuple[str, ...]] = ()
+
+    def __post_init__(self, args: tuple[str, ...] = ()) -> None:
```

Comment:
```suggestion
    def __post_init__(self, args: tuple[str, ...]) -> None:
```

You can drop the default value, `__post_init__` should not be called directly and `dataclasses` will always pass `args`

Reply:
Thanks!